pkg/features/kube_features.go

```diff
@@ -75,6 +75,13 @@ const (
 	// Works only with Docker Container Runtime.
 	Accelerators utilfeature.Feature = "Accelerators"
 
+	// owner: @vishh
+	// alpha: v1.8
+	//
+	// Enables support for Device Plugins
+	// Only Nvidia GPUs are supported as of v1.8.
```

Comment:
Why so? We probably test just GPU plugins. That doesn't mean users cannot install other plugins right?

Reply:
+1

Reply:
Also is the Nvidia plugin ready to go for 1.8?